packages/components/src/templates/next/components/complex/CollectionBlock/utils/getCollectionParent.ts

```diff
@@ -21,7 +21,7 @@ export const getCollectionParent = ({
     return collectionParent as IsomerCollectionPageSitemap
   }
 
-  throw new Error(
-    `CollectionBlock: No collection parent found for collection ID ${collectionId}`,
-  )
+  // NOTE: Signal an error to the caller that no parent could be found
+  // so that we can fail gracefully
```

Comment:
i don't quite understand this comment? 👀

Reply:
we return `null` here rather than throwing an `error`, so that the caller can handle this state by themselves